python/emitter.js

```diff
@@ -265,9 +282,9 @@
             }
         },
         checkWindow(tag, matchTimeout, stitchContent) {
-            let Tag = !tag ? `` : `tag="${tag}"`
-            let MatchTimeout = !matchTimeout ? `` : `,match_timeout=${matchTimeout}`
-            let fully = !stitchContent ? `` : `, fully=${capitalizeFirstLetter(stitchContent)}`
+            let Tag = !tag ? `None` : `tag="${tag}"`
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
            let Tag = !tag ? "None" : `tag="${tag}"`
```